pkg/runtime/ruby/ruby.go

```diff
@@ -152,15 +268,14 @@ func InterpreterInfo(proc procfs.Proc) (*runtime.Interpreter, error) {
 		rubyExecutable = path.Join("/proc/", strconv.Itoa(pid), "/root/", librubyPath)
 	}
 
-	// Read the Ruby version.
-	//
 	// PERF(javierhonduco): Using Go's ELF reader in the stdlib is very
```

Comment:
This code was moved above, no? Could the extracted function be used here?

Reply:
Not, exactly as in current shape. Some of the variables like `symbols` are reused. I don't want to refactor too much code in this PR. We could adapt a similar approach as in `runtime/python` but after some refinements in that code.